CleverTapSDK/InApps/CustomTemplates/SystemAppFunction/CTSystemAppFunctions.h

```diff
@@ -6,17 +6,20 @@
 //  Copyright © 2025 CleverTap. All rights reserved.
 //
 
-#ifndef CTSystemAppFunction_h
-#define CTSystemAppFunction_h
+#ifndef CTSystemAppFunctions_h
+#define CTSystemAppFunctions_h
 
 #import <Foundation/Foundation.h>
 #import "CTCustomTemplate.h"
+#import "CTSystemTemplateActionHandler.h"
 
 @interface CTSystemAppFunctions : NSObject
 
-+ (NSDictionary<NSString *, CTCustomTemplate *> *)systemAppFunctions;
+- (instancetype)initWithSystemTemplateActionHandler:(CTSystemTemplateActionHandler *)systemTemplateActionHandler;
+
+- (NSDictionary<NSString *, CTCustomTemplate *> *)systemAppFunctions;
```

Comment:
Do we need this `CTSystemAppFunctions` as an instance?
The systemAppFunctions can be a static method with the handler:
```
+ (NSDictionary<NSString *, CTCustomTemplate *> *)systemAppFunctionsWithHandler:(CTSystemTemplateActionHandler *)handler {
```

Reply:
Updated changes to use static method with handler.